spring-cloud-gcp-starters/spring-cloud-gcp-starter-config/src/main/java/org/springframework/cloud/gcp/config/GcpConfigProperties.java

```diff
@@ -0,0 +1,59 @@
+/*
+ *  Copyright 2017 original author or authors.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+package org.springframework.cloud.gcp.config;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+
+/**
+ * Configuration for {@link GoogleConfigPropertySourceLocator}.
+ *
+ * @author Jisha Abubaker
+ */
+@ConfigurationProperties("spring.cloud.gcp.config")
```

Comment:
Is the duplication of name and profile from here and `ConfigClientProperties` intended?

Reply:
If we're also using `ConfigClientProperties`, wouldn't it be better to just stick to it for `name` and `profile`?
I noticed that `ConfigClientProperties` doesn't contain a `timeout` field, I think it's fair to use `GcpConfigProperties` just for that bit (since arguably, it's the gcp-specific configuration).